src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -1,47 +1,167 @@
+
 package core.basesyntax;
 
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private static final String INDEX_FOR_EXCEPTION = "Index: ";
+    private static final String SIZE_FOR_EXCEPTION = ", Size: ";
+    private static final int NUMBER_TO_SUBTRACT = 1;
+    private static final int ZERO = 0;
```

Comment:
`NUMBER_TO_SUBTRACT` is not an informative name, it should reflect for what is it used/why exactly this value?
same with `ZERO`